### PR TITLE
Add label color input [WIP]

### PR DESCRIPTION
**Pull request by elenatorro**
Related issue: #13907
Project: CartoDB/product#54

captura de pantalla 2018-05-24 a las 10 21 48
captura de pantalla 2018-05-24 a las 10 22 44

### CODE REVIEW

File: lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-fill-color.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+var NO_PANES_CLASS = 'Editor-formInner--NoTabs';
+
 module.exports = {
   generate: function (params) {
     return {
       type: 'FillColor',
-      title: _t('editor.style.components.labels-fill'),
+      title: _t('editor.style.components.labels-fill-color'),
+      fieldClass: NO_PANES_CLASS,
       query: params.querySchemaModel.get('query'),
```
Comment (VictorVelarde) [inline lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-fill-color.js]: I'm not sure if these extra attributes, such as `query`, are needed anymore in our editors. Maybe we could check the old implementation [here](https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/builder/editor/style/style-form/style-form-components-dictionary.js) against our new approach...

File: lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-halo-color.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+var NO_PANES_CLASS = 'Editor-formInner--NoTabs';
+
+module.exports = {
+  generate: function (params) {
+    return {
+      type: 'FillColor',
+      title: _t('editor.style.components.labels-halo-color'),
+      fieldClass: NO_PANES_CLASS,
+      query: params.querySchemaModel.get('query'),
```
Comment (VictorVelarde) [inline lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-halo-color.js]: same for `halo`

File: lib/assets/javascripts/builder/editor/style/style-form/style-properties-form/style-labels-properties-form-model.js [hunk 3]
```diff
@@ -1,21 +1,38 @@
 var _ = require('underscore');
 var StyleFormDefaultModel = require('builder/editor/style/style-form/style-form-default-model');
 
+var FILL_PROPERTIES = [
```
Comment (VictorVelarde) [inline lib/assets/javascripts/builder/editor/style/style-form/style-properties-form/style-labels-properties-form-model.js]: I like keeping this approach

File: lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-fill-color.js (hunk 1 above)
Comment (elenatorro) [reply lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-fill-color.js]: I know, I totally agree we should review that. Now that we have the new components we need, it is easier to do. I would do that in the test phase, what do you think?

Comment (VictorVelarde) [reply lib/assets/javascripts/builder/editor/style/style-form/style-form-dictionary/labels-fill-color.js]: 👍

File: lib/assets/test/spec/builder/editor/style/style-form/style-form-components-dictionary.spec.js [hunk 4]
```diff
@@ -398,19 +398,21 @@ describe('editor/style/style-form/style-form-components-dictionary', function ()
...
       });
 
       expect(componentDef.editorAttrs).toEqual(jasmine.objectContaining({
-        size: {
-          hidePanes: ['value']
-        },
-        color: {
-          hidePanes: ['value']
-        }
+        hidePanes: ['value']
```
[the remark this answers is not in the document]
Comment (elenatorro) [reply lib/assets/test/spec/builder/editor/style/style-form/style-form-components-dictionary.spec.js]: 👍

**Comment by elenatorro** [comment]
🚀

**Pull request merged**